src/gradebook/views.py

```diff
@@ -38,7 +38,7 @@ def get(self):
             return render_template('studentGradebook.html',
                                    courses=current_user.courses,
                                    tasks=data)
-        elif current_user.permissions == 2:
+        elif current_user.permissions == 50:
```

Comment:
I'm confused with the permissions numbers... can you clarify what the values mean?
I think it's:
admin=100
author=50
TA=20
student=10